### PR TITLE
Synch from partner-center-pr finops

### PR DESCRIPTION
**Pull request by bandersmsft**
## 🛠️ Description

Fixes # 

### 🔬 How did you test this change?

> - [ ] 🤏 Lint tests
> - [ ] 🤞 PS -WhatIf / az validate
> - [ ] 👍 Manually deployed + verified
> - [ ] 💪 Unit tests
> - [ ] 🙌 Integration tests

### 🙋‍♀️ Do any of the following that apply?

> - [ ] 🚨 This is a breaking change.
> - [ ] 🤏 The change is less than 20 lines of code.

### 📑 Did you update `docs/changelog.md`?

> - [ ] ✅ Updated changelog (required for `dev` PRs)
> - [ ] ➡️ Will add log in a future PR (feature branch PRs only)
> - [ ] ❎ Log not needed (small/internal change)

### 📖 Did you update documentation?

> - [ ] ✅ Public docs in `docs` (required for `dev`)
> - [ ] ✅ Internal dev docs in `src` (required for `dev`)
> - [ ] ➡️ Will add docs in a future PR (feature branch PRs only)
> - [ ] ❎ Docs not needed (small/internal change)

### CODE REVIEW

**Comment by bandersmsft** [comment]
@flanakin I moved most images from the higher-level /media folder into corresponding folders and renamed, per MS Learn rule.

File: docs-mslearn/toolkit/power-bi/data-ingestion.md [hunk 1]
```diff
@@ -1,73 +1,80 @@
...
+#customer intent: As a FinOps user, I want to learn about the Data ingestion report so that I can understand my incoming data.
 ---
 
 <!-- markdownlint-disable-next-line MD025 -->
 # Data ingestion report
 
-The **Data ingestion report** provides details about the data you've ingested into your FinOps hub storage account.
-
-You can download the Data ingestion report from the [latest release](https://github.com/microsoft/finops-toolkit/releases/latest).
+The **Data ingestion report** provides details about the data that got ingested into your FinOps hub storage account. You can download the Data ingestion report from the [latest release](https://github.com/microsoft/finops-toolkit/releases/latest).
```
Comment (flanakin) [inline docs-mslearn/toolkit/power-bi/data-ingestion.md:16]:
```suggestion
The **Data ingestion report** provides details about the data that got ingested into your FinOps hub storage account. You can download the Data ingestion report from the [latest release](https://aka.ms/ftk/latest).
```

File: docs-mslearn/toolkit/power-bi/help-me-choose.md [hunk 2]
```diff
@@ -1,87 +1,118 @@
...
+| Accessible outside of Power BI                      |                ✘                 |                ✔                 |                ✔                 |               ✔                |
 | Learn more                                          |  [Learn more][about-connector]   |                                  |     [Learn more][about-hubs]     | [Learn more][about-workspaces] |
 
 [about-connector]: /power-bi/connect-data/desktop-connect-azure-cost-management
 [about-hubs]: ../hubs/finops-hubs-overview.md
 [about-workspaces]: ../../fabric/create-fabric-workspace-finops.md
 [hubs-template]: ../hubs/template.md
 
-_<sup>1) Microsoft Fabric can connect to either raw exports or FinOps hubs. FinOps toolkit reports do not support Microsoft Fabric yet but will in a future release.</sup>_
+_¹ Microsoft Fabric can connect to either raw exports or FinOps hubs. FinOps toolkit reports don't support Microsoft Fabric yet. Development is underway._
```
Comment (flanakin) [inline docs-mslearn/toolkit/power-bi/help-me-choose.md:76]:
```suggestion
_¹ Microsoft Fabric can connect to either raw exports or FinOps hubs. FinOps toolkit reports don't support Microsoft Fabric yet.._
```

File: docs-mslearn/toolkit/power-bi/rate-optimization.md [hunk 3]
```diff
@@ -107,82 +110,97 @@ The **Reservation recommendations** page helps you identify any places where you
...
 
-![Screenshot of the Reservation recommendations page](../../media/power-bi/rate-optimization_reservation-recommendations.png)
+:::image type="content" source="./media/rate-optimization/reservation-recommendations.png" border="true" alt-text="Screenshot of the Reservation recommendations page that shows reservation purchase recommendations." lightbox="./media/rate-optimization/reservation-recommendations.png" :::
 
 <br>
 
 ## Purchases
 
-<!-- NOTE: There is a similar page in the cost-summary.md file. They are not identical. Please keep both updated at the same time. -->
+The **Purchases** page shows any new commitment discount purchases, either monthly or upfront payments, within the specified period.
```
Comment (flanakin) [inline docs-mslearn/toolkit/power-bi/rate-optimization.md:141]:
```suggestion
```

File: docs-mslearn/toolkit/power-bi/rate-optimization.md [hunk 4]
```diff
@@ -107,82 +110,97 @@ The **Reservation recommendations** page helps you identify any places where you
...
```
Comment (flanakin) [inline docs-mslearn/toolkit/power-bi/rate-optimization.md:140]:
```suggestion
```

File: docs-mslearn/toolkit/power-bi/rate-optimization.md [hunk 5]
```diff
@@ -107,82 +110,97 @@ The **Reservation recommendations** page helps you identify any places where you
...
 
-The **Purchases** page shows any new commitment discount purchases (either monthly or upfront payments) within the specified period.
+<!-- NOTE: There is a similar page in the cost-summary.md file. They are not identical. Please keep both updated at the same time. -->
```
Comment (flanakin) [inline docs-mslearn/toolkit/power-bi/rate-optimization.md:142]:
```suggestion
<!-- NOTE: There is a similar page in the cost-summary.md file. They are not identical. Please keep both updated at the same time. -->

The **Purchases** page shows any new commitment discount purchases, either monthly or upfront payments, within the specified period.
```

**Review by flanakin — APPROVED** [review]

**Pull request merged**